developer-docs-site/docs/concepts/basics-merkle-proof.md

```diff
@@ -3,8 +3,14 @@ title: "Proof"
 slug: "basics-merkle-proof"
 ---
 
-The Aptos blockchain uses proofs as a way to verify the authenticity and correctness of blockchain data.
+# Proof
```

Comment:
I kinda feel like ... we shouldn't really even have this section. I don't know who this is targeting.

Reply:
:- ) True. I had the same thought but took a "do no harm first" approach. Another doc qualifies for the same argument. Will clean it up Monday or a day after.

Reply:
I recommend a do harm first approach... or just hit me up on slack and we can agree together to burn it to the ground!